### DISCUSSION TITLE
Terminal Velocity Question

### DISCUSSION THREAD

**Discussion by Aarons-Joshua** [cooperrc/computational-mechanics#20] [Q&A]
Dear Professor Cooper,

I am a little confused how we derived the final velocity formula in the numerical error section of the module. Can you explain further how the final equation was derived? A screenshot is posted with the section.

Thank you!

![image](https://user-images.githubusercontent.com/98834673/171220587-4e71d438-98e9-4a21-8f68-1fbd480ff16a.png)

**Comment by tempcooper** [comment]
Great question @Aarons-Joshua !!

the terminal velocity, $v_{term}$ is the speed when the object stops changing speed i.e.

$\frac{dv}{dt} = 0 = g - \frac{c}{m}v_{term}^2$

so 

$v_{term}^2 = \frac{mg}{c}$

**Reply by Aarons-Joshua** [reply]
I understand that mostly, however how do you get the value v(t) =v_term tanh(gt/v_term)?

Thank you for the help!

**Reply by tempcooper** [reply]
> I understand that mostly, however how do you get the value v(t) =v_term tanh(gt/v_term)?
> 
> Thank you for the help!

Ah, I see. Yes, this is a solution for differential equations. Here is one approach

$\frac{dv}{dt} = g(1 - \frac{c}{mg}v^2)$

and 

$\frac{1}{v_{term}^2} = \frac{c}{mg}$

so 

$\frac{dv}{1-\frac{v^2}{v_{term}^2}} = gdt$

integrating this equation results in 

$v_{term}\tanh^{-1}\left(\frac{v}{v_{term}}\right) = gt$ [wolfram integral](https://www.wolframalpha.com/input?i=int%281%2F%281-a%5E2*x%5E2%29%2C+x%29)

finally, solving for $v$

$\frac{v}{v_{term}} = \tanh\frac{gt}{v_{term}}$

**Reply by Aarons-Joshua** [reply]
Ahh, that makes much more sense! Thank you so much professor!

**Reply by ecieszynski** [reply]
im still a bit confused where the tanh^-1 comes from. Can you clarify a little further?

**Reply by cooperrc** [reply]
> im still a bit confused where the tanh^-1 comes from. Can you clarify a little further?

This step is a just based on a mix of experience/table of derivatives (my table of choice as an undergrad/grad student was Schaums outline series)

$d/dx(\tanh^{-1}ax) = \frac{1}{1-a^2x^2}$

I don't remember off-hamd how to do the power series or complex algebra derivation of this fact though.
